build.sbt

```diff
@@ -411,9 +414,7 @@ lazy val `proxy-core` = (project in file("proxy/core"))
         "io.grpc" % "grpc-netty-shaded" % GrpcJavaVersion,
         // Since we exclude Aeron, we also exclude its transitive Agrona dependency, so we need to manually add it HERE
         "org.agrona" % "agrona" % "0.9.29",
-        // FIXME REMOVE THIS ONCE WE CAN HAVE OUR DEPS (grpc-netty-shaded, agrona, and protobuf-java respectively) DO THIS PROPERLY
-        "org.graalvm.sdk" % "graal-sdk" % GraalVersion % "provided", // Only needed for compilation
-        svmGroupId % "svm" % GraalVersion % "provided", // Only needed for compilation
+        "org.graalvm.nativeimage" % "svm" % GraalVersion % "provided", // Only needed for compilation
```

Comment:
We only need to add this dep since the `graal-sdk` is a transitive dep of this dep.